src/index.html

```diff
@@ -6,9 +6,72 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@800&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="head">
+
+      <a href="index.html" class="head_logo"><img src="images/logo.png" alt="The logo of Moyo"></a>
+
+      <nav class="menu_nav">
+
+        <ul class="head_list">
+
+          <li>
+
+            <a href="#" class="is-active list_link">Apple</a>
+
+          </li>
+
+          <li>
+
+            <a href="#" class="list_link">Samsung</a>
+
+          </li>
```

Comment:
1. a lot of extra empty lines. check recommendations from a checklist
2. use BEM methodology for class naming
3. add a class to `li`
```suggestion
    <header class="header">
      <a href="index.html" class="header__logo">
        <img src="images/logo.png" alt="The logo of Moyo">
      </a>

      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#" class="nav__link is-active">Apple</a>
          </li>

          <li class="nav__item">
            <a href="#" class="nav__link">Samsung</a>
          </li>
```